tests/other_test.go

```diff
@@ -0,0 +1,39 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPlanValidation(t *testing.T) {
+	t.Parallel()
+
+	options := &terraform.Options{
+		TerraformDir: "../examples/basic",
+		Vars: map[string]interface{}{
+			"prefix":         "validate-plan",
```

Comment:
Just a note that if not using the wrapper, the prefix will not have a unique suffix. You can do that easily here like we do in the testwrapper so that you won't have name conflicts.

Reply:
This test is now just using plans so the prefix won't have any conflicts